app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -334,6 +344,27 @@ def parse_persistent_volume(persistent_volume)
       new_result
     end
 
+    def parse_persistent_volume_claim(claim)
+      new_result = parse_base_item(claim)
+      new_result.merge!(
+        :desired_access_modes => claim.spec.accessModes.try(:join, ','),
```

Comment:
I think the right way to do it over here is to create a accessModes model, and have a join table between accessModes and claim (has_many thorugh on both models), although the model is very thin because its only a string.
Another option is binary representation(rwx =7) create something similar with the optional values. 
If there is no validation and it is used only here keeping it as is is also an option.